net.zgcoder.skycheckers.json

```diff
@@ -0,0 +1,136 @@
+{
+	"app-id": "net.zgcoder.skycheckers",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "skycheckers",
+	"finish-args": [
+		"--socket=x11", "--share=ipc",
+		"--device=dri", "--socket=pulseaudio", "--share=network", "--device=all"
+	],
+	"modules": [
+		{
+			"name": "SDL2",
+			"buildsystem": "autotools",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.libsdl.org/release/SDL2-2.0.12.tar.gz",
+					"sha256": "349268f695c02efbc9b9148a70b85e58cefbbf704abd3e91be654db7f1e2c863"
+				}
+			],
+			"cleanup": ["/bin/sdl2-config", "/include", "/lib/libSDL2.la", "/lib/libSDL2main.a", "/lib/libSDL2main.la", "/lib/libSDL2_test.a", "/lib/libSDL2_test.la", "/lib/cmake", "/share/aclocal"]
```

Comment:
Please put an element per line for large arrays/lists like this one & finish-args. It's just unreadable